clms-angular/src/app/modules/courses/modals/select-module-items-dialog/select-module-items-dialog.component.ts

```
@@ -0,0 +1,40 @@
+    TableModule,
+    ButtonDirective
+  ],
+  standalone: true
+})
+export class SelectModuleItemsDialogComponent {
+  items: CourseModuleItemProjection[] = []; // Input list of modules
+  selectedItems: CourseModuleItemProjection[] = []; // Array to hold selected modules
+
+  constructor(public ref: DynamicDialogRef, public config: DynamicDialogConfig<SelectModulesItemsDialogData>) {
+    // Load the modules from config if provided
+    if (config.data && config.data.items) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)